composer.json

```diff
@@ -15,7 +15,7 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^8",
-        "wmde/psr-log-test-doubles": "^2.2"
+        "jeroen/psr-log-test-doubles": "^2.2|^3.0"
```

Comment:
(I moved the location of the library, since I no longer work for wmde.)

Reply:
@JeroenDeDauw i still see activity on https://github.com/wmde/PsrLogTestDoubles
Isn't this kinda double?

Reply:
They merged my changes into their repo after about a month (yesterday). As far as I am concerned `jeroen/psr-log-test-doubles` is the primary location now.